configure.ac

```diff
@@ -1317,15 +1317,15 @@ if test "x$enable_mqtt" = "xyes" || test "x$enable_mqtt" = "xauto"; then
 		CPPFLAGS_SAVE="$CPPFLAGS"
 		LDFLAGS_SAVE="$LDFLAGS"
 		CPPFLAGS="$CPPFLAGS -I$with_libpaho_mqtt/include"
-		LDFLAGS="$LDFLAGS -L$with_libpaho_mqtt -lpaho-mqtt3c"
+		LDFLAGS="$LDFLAGS -L$with_libpaho_mqtt -lpaho-mqtt3cs"
```

Comment:
FYI: On my computer, with local-built paho-mqtt lib this caused that mqtt lib was not found. Turned out, SSL support is disabled by default in paho-mqtt. :)

Reply:
Debian, Ubuntu, Fedora, and Arch AUR enables TLS, I checked them.

Reply:
I've tested it too on Debian Testing :+1.
This was just a small note, in case someone else builds paho-mqtt locally with default flags.